x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -411,6 +411,15 @@ export function App({
         return;
       }
 
+      notifications.toasts.addSuccess(
+        i18n.translate('visualize.topNavMenu.saveVisualization.successNotificationText', {
```

Comment:
As this is a separate plugin, you need a separate i18n id - I suggest `xpack.lens.app.saveVisualization.successNotificationText`

You don't need to add translations manually, this is handled manually

Reply:
Done! Interesting. I was wondering how the translation was done.